### DISCUSSION TITLE
Mitsubishi AC and remote temperature sensor

### DISCUSSION THREAD

**Discussion by host-felice** [arendst/Tasmota#19172] [Q&A]
Hello, I am running 5 wemos d1 mini to drive my Mitsubishi AC. They work great. I have a question about the HVACRemoteTemp command.

It seems it only accepts int numbers like 20 23 27 and so on, so only 1 degree steps. The internal temperature sensor works with 0.5 steps. Anything I write after the . is simply ignored: 22.5 -> 22. It is not rounded because 22.9 is still 22.

How can I use the 0.5 steps?

Thank you very much.

**Comment by sfromis** [comment]
As the driver code is using an integer for the command, I see no current option for decimals.

**Reply by host-felice** [reply]
Thank you for your reply. Isn't possible to change the driver code to allow for 0.5 step? The AC should be fine with it I suppose since the internal temperature sensor works with 0.5 step

**Reply by sfromis** [reply]
Looks like it would be *possible* to someone wanting to change the [driver code](https://github.com/arendst/Tasmota/blob/development/tasmota/tasmota_xdrv_driver/xdrv_44_miel_hvac.ino#L807-L849), as it ends with adding an offset and multiplying by 2.

**Reply by host-felice** [reply]
I have no idea how to do it, but I can test the mod if someone is willing to change the code :-)
